tests/AuthTest.php

```diff
@@ -192,63 +206,157 @@ public function testDeleteProfileNotLoggedIn()
 
             $this->assertResponseStatus(403);
         }
-        
-        public function testDelete()
+
+        public function testUserNotFound()
+        {
+
+            $this->json(
+                'GET',
+                'api/v1/app/starapi-testing/profiles/2343423',
+                [],
+                [
+                    'Authorization' => $this->testValidLogin()
```

Comment:
store auth header value within class property and fetch that instead of repeating login test